src/server/v2.0/handler/preheat.go

```diff
@@ -22,6 +23,7 @@ func newPreheatAPI() *preheatAPI {
 	return &preheatAPI{
 		preheatCtl: preheatCtl.Ctl,
 		projectCtl: projectCtl.Ctl,
+		enforcer:   preheatCtl.NewEnforcer(),
```

Comment:
How about add a global enforcer like other controllers？

Reply:
`mmpei` add a global enforcer at  #12428, I'll rebase and use it.